dotcom-rendering/src/components/Nav/ExpandedMenu/CollapseColumnButton.tsx

```diff
@@ -102,10 +102,7 @@ export const CollapseColumnButton = ({
 		// eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role -- we’re using this label for a CSS-only toggle
 		role="menuitem"
 		data-cy={`column-collapse-${title}`}
-		data-link-name={nestedOphanComponents(
-			'nav2',
-			`column-toggle-${title}: show`,
-		)}
+		data-link-name={nestedOphanComponents('nav2', `secondary`, title)}
```

Comment:
Was added in https://github.com/guardian/dotcom-rendering/pull/2758 but I don't think the value is still valid. I can't find it in frontend DOM and the script says the following are missing:

nav2 : secondary : News → <BUTTON/>
nav2 : secondary : Opinion → <BUTTON/>
nav2 : secondary : Sport → <BUTTON/>
nav2 : secondary : Culture → <BUTTON/>
nav2 : secondary : Lifestyle → <BUTTON/>